src/entities/dataset/measure-row.ts

```diff
@@ -0,0 +1,58 @@
+import { Entity, PrimaryColumn, Column, BaseEntity, ManyToOne, JoinColumn } from 'typeorm';
+
+import { DisplayType } from '../../enums/display-type';
+
+import { Measure } from './measure';
+
+@Entity({ name: 'measure_rows' })
```

Comment:
Singular please.

Reply:
I'll change it but I do object... These are collections of things.  They don't stand on their own in the same way as a dataset or revision might.  You need all of the collection.  Typically in programming we use the singular because we have classes and array identifiers which don't exist in the Databases.  Typically a table is considered a collection of things and is plural... Which is important when you have a collection of collections such as FactTableColumns or MeasureRows.